templates/docs/base/code.md

```diff
@@ -54,26 +54,11 @@ View example of the code snippet
 
 ### Copyable
 
-This component should be used when displaying a single line of code, accompanied by a copy icon, which allows users to copy the provided code to their clipboard.
-
-<div class="p-strip is-shallow">
-  <div class="row">
-     <div class="col-4">
-       <div class="p-notification--positive">
-        <p class="p-notification__response"><span class="p-notification__status">Do:</span>Use for single line terminal commands, functions or instructions.</p>
-       </div>
-     </div>
-    <div class="col-4">
-      <div class="p-notification--negative">
-        <p class="p-notification__response"><span class="p-notification__status">Don't:</span>Use for multiline code. If needed, code <a href="#block" class="p-notification__action">block</a> should be used.</p>
-      </div>
-    </div>
-  </div>
-</div>
-
-<div class="embedded-example"><a href="/docs/examples/patterns/code-copyable/" class="js-example">
-View example of the code copyable pattern
-</a></div>
+<span class="p-label--deprecated">Deprecated</span>
+
+The code copyable element is now deprecated and will be removed in a future version of Vanilla.
+
+It is being replaced by the new [code snippet pattern](/docs/base/code#code-snippets).
 
 ### Functionality
```

Comment:
While we are here, this seems to be a sub-section of 'Copyable', so should be h4 I guess.

```suggestion
#### Functionality
```

Reply:
Since we're deprecating this pattern, I should remove this section right? I overlooked it previously, but "the example" it mentions is no longer linked to.